Changelog_Template.md

```diff
@@ -0,0 +1,46 @@
+This is the template file for changelogs. See the following
+[Confluence page](https://zombie.atlassian.net/wiki/spaces/PRODUCT/pages/2040136662/Create+Changelog+Entries) for the detailed procedure.
+
+TLDR, the following template includes _sample_ entries that you can modify/include
+in the `content/en/Changelog/_index.md` file.
+
+1. If you're including API changes, find the version
+1. Substitute existing changes in each category.
+   1. If there are no changes in a category, delete the title (such as ### Added).
+1. Create a new branch.
+1. Insert the changes in the `content/en/Changelog/_index.md` file, just before the next oldest changelog entry.
+1. Create a PR.
+1. Assign the PR **review** to a member of the [External Product Documentation]() team. If none are available, 
```

Comment:
The last step seems to be incomplete as the link doesn't have a target and the last sentence is unfinished.

I would also suggest merging these two steps together to shorten the list of instructions. At first sight, the procedure will seem easier to follow. :slightly_smiling_face:

```suggestion
1. Create a PR and assign the **review** to a member of the [External Product Documentation]() team. If none are available, [...]
```